File: python/7 kyu/79.py
# https://www.codewars.com/kata/55cb000321ca31039e00007d/train/python

# Passed

def geometric_sequence_sum(a, r, n):
    sequence_sum = a
    current_term = a
    for _ in range(1, n):
        term_value = r * current_term
        current_term = term_value
        sequence_sum += term_value

    return sequence_sum

output = geometric_sequence_sum(2, 3, 5)
print(output)

File: nodejs/7 kyu/79.js
// https://www.codewars.com/kata/567d71b93f8a50f461000019/train/javascript

// Passed

const crossover = (chromosome1, chromosome2, index) => {
  let crossover1 = chromosome1.slice(index);
  let crossover2 = chromosome2.slice(index);
  let crossoveredChromosomes = [
    chromosome1.slice(0, index) + crossover2,
    chromosome2.slice(0, index) + crossover1,
  ];
  return crossoveredChromosomes;
};

const output = crossover('111000','000110',3);
console.log(output);